components/script/dom/globalscope.rs

```diff
@@ -78,6 +79,9 @@ pub struct GlobalScope {
     crypto: MutNullableDom<Crypto>,
     next_worker_id: Cell<WorkerId>,
 
+    /// A place to keep the clones read into this realm rooted.
+    structured_clone_holder: DomRefCell<Vec<DomRoot<Blob>>>,
```

Comment:
This can store `Dom<Blob>` instead of using DomRoot, since it's a field inside of a DOM object that the GC already knows about.